<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AjoutTacheWindow</class>
 <widget class="QDialog" name="AjoutTacheWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>277</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ajout d'une Tâche</string>
  </property>
  <widget class="QSplitter" name="splitter_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>261</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QSplitter" name="splitter">
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="Projet_Lab">
     <property name="text">
      <string>Projet</string>
     </property>
    </widget>
    <widget class="QComboBox" name="Projet_Box"/>
   </widget>
   <widget class="QSplitter" name="splitter_2">
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="ID_Lab">
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ID_Edit"/>
    <widget class="QCheckBox" name="Preemptee_Box">
     <property name="text">
      <string>Préemptée</string>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_6">
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="Titre_Lab">
     <property name="text">
      <string>Titre</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Titre_Edit"/>
   </widget>
   <widget class="QSplitter" name="splitter_5">
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="Dispo_Lab">
     <property name="text">
      <string>Date de disponibilité</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="Dispo_Edit">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_3">
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="Echeance_Lab">
     <property name="text">
      <string>Date d'échéance</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="Echeance_Edit">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_4">
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="Duree_Lab">
     <property name="text">
      <string>Durée</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="Heure_Box"/>
    <widget class="QLabel" name="Heure_Lab">
     <property name="text">
      <string>Heure(s)</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="Minute_Box"/>
    <widget class="QLabel" name="Minute_Lab">
     <property name="text">
      <string>Minute(s)</string>
     </property>
    </widget>
   </widget>
   <widget class="QDialogButtonBox" name="buttonBox">
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AjoutTacheWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AjoutTacheWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
